system/Database/MySQLi/Connection.php

```diff
@@ -483,14 +483,14 @@ protected function _foreignKeyData(string $table): array
     {
```

Comment:
> Fix MySql _foreignKeyData() SQL upper case 

Please write **why** you changed to uppercase in the commit message.

We can easily know that this commit changed the column names to uppercase,
but it is difficult to know why this change was needed from the commit.